### DISCUSSION TITLE
Componentes no composables pero igual de importantes

### DISCUSSION THREAD

**Discussion by 006900** [ArisGuimera/JetpackComposePro#47] [Ideas]
bien la idea va de que la mayoria de los composables los usamos todos ya que los proporciona jetpack compose pero que tal si creamos un apartado de otros componentes que no sean precisamente de compose ej:la libreria de accompanist nos permite ocultar facilmente la toolbar en compose o como el modo oscuro que no tiene nada que ver pero estaria bueno que lo explicaran en los videos , temas y estilos . Que piensan al respecto?

**Comment by ArisGuimera** [comment]
Me parece una buena idea, tenemos que buscar en qué sección ponerlo o como organizarlo. Vamos a ver que propone la comunidad :)

**Reply by 006900** [reply]
yo propondria que crearamos un nuevo apartado llamado : UI Features,
UI Schemes o Ui Model donde mostremos como hacer correctamente ej:Splash Screen o como ocultar el top bar y otras caracteristicas que nos ayudarian a mejorar nuestras ui en compose

**Reply by ArisGuimera** [reply]
Voy a darle una pensada a ver!
